README.md

```diff
@@ -37,14 +37,22 @@ bucket.company("acme_inc", { name: "Acme Inc", plan: "pro" }, "john_doe");
 // track events
 bucket.track("sent_message", { foo: "bar" }, "john_doe", "company_id");
 
-// collect qualitative feedback
+// send qualitative feedback
 bucket.feedback({
   featureId: "my_feature_id",
   userId: "john_doe",
   companyId: "acme_inc", // String (optional)
   score: 5, // Number: 1-5 (optional)
   comment: "Absolutely stellar work!", // String (optional)
 });
+
+// collect qualitative feedback using built-in dialog
+bucket.collectFeedback({
```

Comment:
quick drive-by: let's call this `openFeedbackForm` or `openFeedbackDialog` to make it super explicit. `collectFeedback` could mean _send_ the feedback.